syntax = "proto3";

message InventoryItem {
    int32 id = 1;
    int32 product_id = 2;
    int32 variant_id = 3;
    int32 quantity = 4;
    string status = 5;
}

